src/NavigationWorkflow.php

```diff
@@ -22,19 +24,22 @@
  * @see      /src/models/Node.php
  */
 class NavigationWorkflow {
+    public $Tree;
+    public function __construct() {
+        $this->Tree = G::build(ArboristWorkflow::class);
+        $this->Tree->setRoot('/sites/'.$_SERVER['SERVER_NAME']);
```

Comment:
Would it be better to receive an Arborist from the calling Controller?

Reply:
Figured it would be easier to just build it in __construct than to pass it every time.